sdk/src/openapi/builder/openapi.builder.ts

```diff
@@ -10,15 +10,20 @@ export function buildOpenApiSchema(version: string, env: string, publicUrl: stri
     openapi: "3.1.0",
     info: {
       title:
-        lang === "fr" ? "Documentation technique de l'API Apprentissage" : "API Apprentissage technical documentation",
+        lang === "fr"
+          ? "Documentation technique de l'espace développeurs La bonne alternance"
+          : "Developer Space La bonne alternance technical documentation",
       version,
       license: {
         name: "Etalab-2.0",
         url: "https://github.com/etalab/licence-ouverte/blob/master/LO.md",
       },
       termsOfService: "https://api.apprentissage.beta.gouv.fr/cgu",
       contact: {
-        name: lang === "fr" ? "Équipe API Apprentissage" : "API Apprentissage team",
+        name:
+          lang === "fr"
+            ? "Équipe Espace développeurs La bonne alternance"
+            : "Espace développeurs La bonne alternance team",
```

Comment:
`The "La bonne alternance" developer space team` ?

Reply:
👉🏼 https://github.com/mission-apprentissage/api-apprentissage/pull/336/commits/71f1a0c1a5091c48c0f67f6a852824672d12d58b